website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -13,6 +13,15 @@ upgrade. However, specific versions of Nomad may have more details provided for
 their upgrades as a result of new features or changed behavior. This page is
 used to document those details separately from the standard upgrade flow.
 
+## Nomad 1.1.4 and 1.2.0
+
+#### Log file names
+
+The [`log_file`] configuration option was not being fully respected, as the
+generated filename would include a timestamp. After upgrade, the active log
+file will always be the value defined in `log_file`, with timestamped files
+being created during log rotation.
```

Comment:
> ...with timestamped files being created during log rotation.

Maybe be even more explicit:

> Timestamped files will only be created if \[log rotation\]\(...\) is configured.

Reply:
I think log rotation is always enabled. The controls you have are `log_rotate_bytes` and `log_rotate_duration`, which defaults to `24h`. 

You can set `log_rotate_bytes` to `0` to not rotate on size. If you set `log_rotate_duration` to `0` you will have a bad time 😬

Reply:
Yet another reason why I need to run more long lived clusters. Thanks! :sweat_smile: